1_exe/index.php

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World</title>
+    <link rel="icon" type="image/x-icon" href="pol.ico">-
+</head>
+<body>
+    <?php
+
+echo "Hello World";
```

Comment:
Please refer to PSR-1 and PSR-12 for proper indention.

Check instruction for exercise 1 for output format.